### PR TITLE
Deprecated playbooks

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34896

## Description
- Add "deprecated" field to all deprecated playbooks
- Add "Deprecated. " at the beginning of related README.md files
- update release version and metadata files

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [X] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

File: Packs/CommonPlaybooks/ReleaseNotes/1_9_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Dedup - Generic v2
+Deprecated playbboks were updated with "deprecated" field
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/ReleaseNotes/1_9_5.md:4]: @tomer-pan typo

Comment (tomer-pan) [reply Packs/CommonPlaybooks/ReleaseNotes/1_9_5.md]: Fixed @David-BMS

**Review by David-BMS — COMMENTED** [review]
@tomer-pan 
Please where possible use this syntax where the replacement playbook is known

Deprecated. Use the Dedup Generic v3 playbook instead.

**Comment by tomer-pan** [comment]
> @tomer-pan
> Please where possible use this syntax where the replacement playbook is known
> 
> Deprecated. Use the Dedup Generic v3 playbook instead.

@David-BMS  Done.

File: Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment.yml [hunk 2]
```diff
@@ -165,3 +166,4 @@ inputs: []
 outputs: []
 tests:
 - No tests - Deprecated playbook
```
Comment (David-BMS) [inline Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment.yml:168]: @tomer-pan 
Keep the same format
tests:
- no test

File: Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_README.md [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+| Account.Username | The Account username | string |
+| Account.Email | The email address associated with the Account | unknown |
+| Account.Groups | The groups the Account is part of | unknown |
+| Account.DisplayName | The Account display name | string |
+| Account.Manager | The Account's manager | string |
 
 ## Playbook Image
 ---
-![Account_Enrichment_Generic](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Account_Enrichment_Generic.png)
+![Account Enrichment - Generic](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_README.md:42]: @tomer-pan you deleted the image

File: Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_v2_README.md [hunk 4]
```diff
@@ -41,4 +38,4 @@ This playbook does not use any commands.
 
 ## Playbook Image
 ---
-![Account_Enrichment_Generic_v2](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Account_Enrichment_Generic_v2.png)
+![Account Enrichment - Generic v2](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_v2_README.md:41]: @tomer-pan deleted image

File: Packs/DeprecatedContent/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_README.md [hunk 5]
```diff
@@ -16,44 +16,44 @@ This playbook does not use any sub-playbooks.
...
+| File.xapmm | The PDF File xapmm | unknown |
+| File.ModDate | The PDF File ModDate | unknown |
+| File.CreationDate | The PDF File CreationDate | unknown |
+| File.Pages | Number of pages in PDF file | unknown |
+| URL.Data | List of URLs that were extracted from the PDF file | unknown |
 
 ## Playbook Image
 ---
-![Extract_Indicators_From_File_Generic](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Extract_Indicators_From_File_Generic.png)
+![Extract Indicators From File - Generic](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/DeprecatedContent/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_README.md:59]: @tomer-pan Deleted image

File: Packs/DeprecatedContent/Playbooks/playbook-QRadar_-_Get_offense_correlations__README.md [hunk 6]
```diff
@@ -26,42 +20,42 @@ This playbook does not use any commands.
...
+| QRadar.Log.IdentityIP | The log's identity IP | unknown |
+| QRadar.Log.Username | The log's username | unknown |
+| QRadar.Log.StartTime | The log's start time | unknown |
+| QRadar.Log.Magnitude | The log's magnitude | unknown |
+| QRadar.Log.ProtocolName | The log's protocol name | unknown |
 
 ## Playbook Image
 ---
-![QRadar_Get_offense_correlations](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/QRadar_Get_offense_correlations.png)
+![QRadar - Get offense correlations](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/DeprecatedContent/Playbooks/playbook-QRadar_-_Get_offense_correlations__README.md:61]: @tomer-pan Deleted image

File: Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment.yml (hunk 2 above)
Comment (tomer-pan) [reply Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment.yml:168]: @David-BMS I don't think that this is something that I changed - it was there already. Also, there are a lot of different formats across all playbooks. I can change this one though, no problem. I do think that it's not worth the time to change other deprecated playbooks to match this format.

File: Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_README.md (hunk 3 above)
Comment (tomer-pan) [reply Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_README.md]: @David-BMS Done (including all of the other md files which the image was deleted in)

**Review by David-BMS — APPROVED** [review]

**Pull request merged**